starfish/src/main/java/org/maxandersen/Starfish.java

```diff
@@ -0,0 +1,238 @@
+package org.maxandersen;
+
+import io.quarkus.runtime.QuarkusApplication;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Objects;
+import java.io.File;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+
+public class Starfish implements QuarkusApplication {
+  @Override
+  public int run(String... args) throws Exception {   
+    
+    
+    for(String s:args)
+    System.out.println("\nTarget : "+s);//Printing Received Arguements for Debugging
```

Comment:
Either remove such debug stuff or use a logger.

Reply:
Initially, I used it for debugging but lately, I thought it fine to display the repository info which user is cloning so I left it as it is.Will remove now

Reply:
have commented it out